dist/src/main/java/io/camunda/application/initializers/WebappsConfigurationInitializer.java

```diff
@@ -34,7 +35,7 @@ public class WebappsConfigurationInitializer
   private static final Set<String> WEBAPPS_PROFILES =
       Set.of(OPERATE.getId(), TASKLIST.getId(), IDENTITY.getId());
   private static final Set<String> LOGIN_DELEGATED_PROFILES =
-      Set.of(IDENTITY_AUTH.getId(), SSO_AUTH.getId());
+      Set.of(IDENTITY_AUTH.getId(), SSO_AUTH.getId(), AUTH_BASIC.getId());
```

Comment:
@Ben-Sheppard I am trying to understand the difference between `AUTH` , `AUTH_BASIC` and `IDENTITY_AUTH`
is `AUTH_BASIC` considered as authentication handled by (delegated to) identity with some basic functionalities? 
How `Identity` webapp behaves when `AUTH` profile is used?

Reply:
Ah good point, so in this case its I guess not delegated in that sense to Identity, the `AUTH_BASIC` profile relies currently on the Spring basic authentication login (the browser popup) so actually in that sense I don't think its considered delegated i.e. to the Identity service.

I will remove it from this set